src/test.zig

```diff
@@ -41,6 +41,8 @@ test {
     testing.refAllDecls(@import("parse"));
     testing.refAllDecls(@import("parse/test/ast_node_store_test.zig"));
 
+    testing.refAllDeclsRecursive(@import("compile"));
```

Comment:
@lukewilliamboswell Does this look necessary/redundant given the module changes? I started this branch before they landed.

Reply:
I would expect that to crash with the same issue as https://roc.zulipchat.com/#narrow/channel/395097-compiler-development/topic/casual.20conversation/near/531379992

Similar to base we probably need to pull all the unit tests out into a separate file `compile_test.zig` and import that in `src/test.zig` with `testing.refAllDecls`